code/modules/surgery/dental_implant.dm

```diff
@@ -42,10 +42,10 @@
 	)
 	return ..()
 
-/datum/action/item_action/hands_free/activate_pill
+/datum/action/item_action/activate_pill
 	name = "Activate Pill"
-
-/datum/action/item_action/hands_free/activate_pill/Trigger(trigger_flags)
+	check_flags = NONE
+/datum/action/item_action/activate_pill/Trigger(trigger_flags)
```

Comment:
This also means it's usable while dead which is probably not idea

You can get around that with:
```dm
/datum/action/item_action/activate_pill/IsAvailable(feedback)
    if(owner.stat > SOFT_CRIT)
        return FALSE
    return ..() 
```